test_rigs/parsing-tests/unit-tests.ref

```diff
@@ -29538,6 +29540,27 @@
             "    (s_username",
             "      USERNAME:'username'  <== mode:DEFAULT_MODE",
             "      (variable",
+            "        VARIABLE:'demo'  <== mode:DEFAULT_MODE)",
+            "      (u",
+            "        (u_privilege",
+            "          PRIVILEGE:'privilege'  <== mode:DEFAULT_MODE",
+            "          (variable",
+            "            DEC:'15'  <== mode:DEFAULT_MODE)))",
+            "      (u",
+            "        (u_nohangup",
+            "          NOHANGUP:'nohangup'  <== mode:DEFAULT_MODE))",
+            "      (u",
+            "        (u_password",
+            "          PASSWORD:'password'  <== mode:DEFAULT_MODE",
+            "          (up_cisco",
+            "            DEC:'0'  <== mode:DEFAULT_MODE",
+            "            (variable_secret",
+            "              VARIABLE:'demo'  <== mode:DEFAULT_MODE))))",
+            "      NEWLINE:'\\n'  <== mode:DEFAULT_MODE))",
+            "  (stanza",
+            "    (s_username",
+            "      USERNAME:'username'  <== mode:DEFAULT_MODE",
+            "      (variable",
```

Comment:
This portion of the diff is problematic. It has nothing to do with your change, and may introduce a conflict. Can you amend this last commit and remove this, then force push?
Assuming you are in the root of your repo on the last commit, do:
1. Save the commit message somewhere so you don't lose it.
2.
```
git reset --soft HEAD^
git reset -p   #### Here you remove the offending portion that is unrelated to juniper_route_filter
git commit ### Here you re-enter the saved commit message
git push -f
```
If tests don't pass after this, you could then try updating the branch by merging from master. But let me take a look first after you've taken the above steps.